internal/pkg/plugin/argocd/delete.go

```diff
@@ -28,7 +28,7 @@ func Delete(options *map[string]interface{}) (bool, error) {
 		return false, err
 	}
 
-	log.Info("uninstalling argocd helm chart")
+	log.Info("Uninstalling argocd helm chart")
```

Comment:
Missing "period" (.) at the end of the sentence.

Uninstalling -> Deleting (to be consistent with our interface's name)